app/(app)/(purchase-and-payment)/ticket-purchase.tsx

```diff
@@ -49,26 +47,23 @@ const TicketPurchasePage = () => {
         </Link>
       }
     >
-      {isPending ? (
-        // TODO show this content when ticket status is PENDING
+      {isPending || data?.paymentStatus === 'PENDING' ? (
         <ContentWithAvatar title="Ticket is being processed" text={ticketId} />
-      ) : isError ? (
-        // TODO show tis content when ticket status is FAIL
-        <ContentWithAvatar variant="error" title={t('paymentFailed')} text={t('paymentFailedText')}>
-          <Panel className="bg-negative-light">
-            <Typography>{error.message}</Typography>
-          </Panel>
-        </ContentWithAvatar>
-      ) : (
-        // TODO show this content when ticket status is SUCCESS
+      ) : data?.paymentStatus === 'SUCCESS' ? (
         <ContentWithAvatar
           variant="success"
           title={t('paymentSuccessful')}
           text={t('paymentSuccessfulText')}
         >
           <BoughtTicket ticket={data} />
         </ContentWithAvatar>
-      )}
+      ) : isError || data.paymentStatus === 'FAIL' ? (
```

Comment:
Please follow the standard order of states if possible:
https://tanstack.com/query/latest/docs/react/guides/queries#query-basics